code/modules/mob/observer/eye/eye.dm

```diff
@@ -133,3 +142,32 @@
 	else
 		sprint = initial
 	return 1
+
+// Used to apply fullscreen effects etc. to owner
+/mob/observer/eye/apply_visual(var/mob/M)
+	if(M != owner || !M.client)
+		return FALSE
+	return TRUE
+
+/mob/observer/eye/remove_visual(var/mob/M)
+	if(M != owner || !M.client)
+		return FALSE
+	return TRUE
+
+/mob/observer/eye/ClickOn(var/atom/A, var/params)
+	if(owner)
+		return ..()
+	
+	return owner.ClickOn(A, params)
```

Comment:
`owner` is either null in which case till will runtime, or line 158-159 will return first